### PR TITLE
[CI] Publish tutorials tarball asset on releases

### PR DESCRIPTION
**Pull request by daniels290813**
Adding update-tutorials step to release tutorials tar

### CODE REVIEW

**Review by liranbg — APPROVED** [review]
👏🏼

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -157,3 +157,19 @@ jobs:
           allowUpdates: true
           prerelease: ${{ github.event.inputs.pre_release }}
           body: ${{ steps.resolve-release-notes.outputs.body }}
+          
+  update-tutorials:
```
Comment (liranbg) [inline .github/workflows/release.yaml:161]: you need to wait for `create-releases` to finish.
add 
`needs: create-releases`

**Review by liranbg — COMMENTED** [review]
here

**Comment by daniels290813** [comment]
@liranbg Thanks !

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**